packages/web/src/components/typography/body.tsx

```diff
@@ -0,0 +1,47 @@
+import styled, { createGlobalStyle, css } from 'styled-components';
+
+// TODO
+// - Spacing
+// - Links with state
+
+// darkFont defines font color
+// Most of the app in on dark background and thus has bright font
+// If text should be black/dark
+
+// Articles, longer text
+const Body1 = styled.p<{ darkFont: boolean }>`
+  color: ${(props) =>
+    props.darkFont ? props.theme.colors.black.primary : props.theme.colors.white.primary};
+  fontsize: ${(props) => props.theme.fontSizes.body1.size};
+  lineheight: ${(props) => props.theme.fontSizes.body1.leading};
+  fontstyle: normal;
+  fontweight: ${(props) => props.theme.fontSizes.body1.weight};
+`;
+// Boxes, shorter text
+const Body2 = styled.p<{ darkFont: boolean }>`
+  color: ${(props) =>
+    props.darkFont ? props.theme.colors.black.primary : props.theme.colors.white.primary};
```

Comment:
When there is more then one prop destructuring is also working just fine: 
```suggestion
  color: ${({ darkFont, theme }) =>
   darkFont ? theme.colors.black.primary : theme.colors.white.primary};
```